### ISSUE TITLE
Issue during install.

### ISSUE THREAD

**Issue by DougHayward** [feathers-nuxt/template-app#2]
# Issue
According to the documentation install should be done with:

```
sao @feathers-nuxt/template-app --update awesome-app
```

This causes the following error:
```
$ sao @feathers-nuxt/template-app awesome-app --update
error Response code 404 (Not Found)
github.com/@feathers-nuxt/template-app/archive/master.zip
```
I thought removing the **@** may help, which it does in a fashion however a new error occurs:

```
$ sao feathers-nuxt/template-app --update awesome-app
success Downloaded repo
success Installed packages
error ReferenceError: ejs:86
    84|     },
    85|     head: {
 >> 86|       title: '<%= name %>',
    87|       meta: [
    88|         { charset: 'utf-8' },
    89|         { name: 'viewport', content: 'width=device-width, initial-scale=1' },

name is not defined
    at eval (eval at compile (/usr/lib/node_modules/sao/node_modules/ejs/lib/ejs.js:618:12), <anonymous>:11:26)
    at Object.returnedFn [as fn] (/usr/lib/node_modules/sao/node_modules/ejs/lib/ejs.js:653:17)
    at Transformer.render (/usr/lib/node_modules/sao/node_modules/jstransformer/index.js:289:25)
    at fileList.forEach.relativePath (/usr/lib/node_modules/sao/lib/runActions.js:64:27)
    at Array.forEach (<anonymous>)
    at stream.use (/usr/lib/node_modules/sao/lib/runActions.js:57:20)
    at current.then (/usr/lib/node_modules/sao/node_modules/majo/dist/majo.cjs.js:58:49)
    at <anonymous>

```
# System spec
- NPM -v: 6.4.1
- Node -v: 8.12.0

**Comment by kharhys** [comment]
Indeed the issue was caused by breaking changes made on sao.js. I have updated the template to be compatible with the latest version of sao.js which fixed both issues.

Initially `sao @feathers-nuxt/template-app --update awesome-app` would download the template from npm while `sao feathers-nuxt/template-app --update awesome-app` *(without the @)* would download the same from github. 

Following the changes on sao.js, installation from npm should be done by calling `sao npm:@feathers-nuxt/template-app --update awesome-app` *(note the prefix npm:)*. Installation from github remains the same as before. 

I updated the guide to reflect this. I also updated all dependencies to their most recent versions. The updates fixed this issue except that now whenever I include `favicon.ico` file in the template, I get a ejs error similar to one  above. I'm looking further into it, but in the meantime the template is usable as is.
